src/jit/morph.cpp

```diff
@@ -16834,15 +16834,21 @@ void Compiler::fgMorph()
     fgDebugCheckBBlist(false, false);
 #endif // DEBUG
 
+    EndPhase(PHASE_MORPH1);
+
     /* Inline */
     fgInline();
 #if 0
     JITDUMP("trees after inlining\n");
     DBEXEC(VERBOSE, fgDispBasicBlocks(true));
 #endif
 
+    EndPhase(PHASE_MORPH2);
+
     RecordStateAtEndOfInlining(); // Record "start" values for post-inlining cycles and elapsed time.
 
+    EndPhase(PHASE_MORPH3);
```

Comment:
I'm not very familiar with what this does, but I would be surprised if it's significant - on the other hand, I guess we can always fold it back in with inlining if it's not useful.